ui/components/component-library/container/README.mdx

```diff
@@ -19,7 +19,7 @@ Using the `maxWidth` prop with `ContainerMaxWidth` enum values, you can set the
 
 Possible values are:
 
-- `ContainerMaxWidth.Sm` (max-width: 320px)
+- `ContainerMaxWidth.Sm` (max-width: 360px)
```

Comment:
Updating incorrect pixel value for max width it should align with ModalContent sizes and it's also specified in the ticket https://github.com/MetaMask/metamask-extension/issues/22076
<img width="474" alt="Screenshot 2024-01-04 at 3 50 55 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/9ae76133-fe23-4997-bcf4-dd2602b34aeb">

Made sure to update all instances of 320px

![Screenshot 2024-01-04 at 3 33 33 PM](https://github.com/MetaMask/metamask-extension/assets/8112138/525e8836-8429-448b-b3c1-6b2635281878)